docs/4.3/enterprise/quickstart-enterprise.md

```diff
@@ -315,6 +320,101 @@ Any SAML-compliant provider can be configured with Teleport by following the
 same steps.  There are Teleport Enterprise customers who are using Oracle IDM,
 SailPoint and others.
 
+## Run Teleport Enterprise using Docker
+
+We provide pre-built Docker images for every version of Teleport Enterprise. These images are hosted on quay.io.
+
+- [All tags under `quay.io/gravitational/teleport-ent` are Teleport Enterprise images](https://quay.io/repository/gravitational/teleport-ent?tag=latest&tab=tags)
```

Comment:
nit: should the link be this whole sentence or just the image name?

Reply:
I think I was going for making it very obvious that it was a clickable link, as code blocks have different formatting.